### DISCUSSION TITLE
How to remove a project

### DISCUSSION THREAD

**Discussion by DJultrakid** [vercel/community#2839] [Help] [answered]
### Summary

So I messed up on my project and now I can't delete it. Can someone tell me how?

### Steps to Reproduce

There isn't a way to replicate the problem I just put a space in a file when there should've been a -

**Comment by smaeda-ks** [comment, marked as answer]
@DJultrakid please refer to the following documentation:
https://vercel.com/docs/concepts/projects/overview#deleting-a-project

you can also remove individual deployments as well:
https://vercel.com/guides/how-do-i-delete-an-individual-deployment
